clang/test/CXX/drs/cwg1xx.cpp

```diff
@@ -922,6 +922,49 @@ namespace cwg155 { // cwg155: dup 632
   // expected-warning@-1 {{braces around scalar initializer}}
 }
 
+namespace cwg156 { // cwg156: partial
+namespace ex1 {
+struct A {
+  operator int();
+} a;
+void foo() {
+  typedef int T;
+  a.operator T(); // T is found using unqualified lookup
+                  // after qualified lookup in A fails.
+}
+} // namespace ex1
+
+namespace ex2 {
+struct A {
+  typedef int T;
+  operator T();
+};
+struct B : A {
+  operator T();
+} b;
+void foo() {
+  b.A::operator T(); // FIXME: qualified lookup should find T in A.
+  // expected-error@-1 {{unknown type name 'T'}}
```

Comment:
This seems to be #28181
Do we want to take a crack at it?

Reply:
It is similar to that issue in a sense that `T` needs to undergo a qualified lookup, but the lookup context is different (nested name specifier here, type of the object expression there).

> Do we want to take a crack at it?

Maybe, but not in this PR.